### PR TITLE
Changes to build on latest uavcan master with FW upload and Node ID

### PR DESCRIPTION
**Pull request by davids5**
allocation

UPDATE: It is OK to merge including the 8f032a6 

See https://github.com/PX4/Firmware/commit/ec1b77c9e1fcca6c0984539d72e8c324f063a624

The recent changes ec1b77c9e1fcca6c0984539d72e8c324f063a624 to parms.h affecting alignment

### CODE REVIEW

**Comment by davids5** [comment]
FYI @pavel-kirienko , @LorenzMeier 

Memory use on master with this PR
```
nsh> free
             total       used       free    largest
Mem:        223200     136624      86576      84544
nsh> uavcan start
uavcan: Node ID 1, bitrate 1000000
uavcan: SW version vcs_commit: 0xd720a42a
uavcan: sensor bridge 'gnss' init ok
uavcan: sensor bridge 'mag' init ok
uavcan: sensor bridge 'baro' init ok
nsh> free
             total       used       free    largest
Mem:        223200     175136      48064      43712
```

File: src/modules/uavcan/sensors/baro.cpp [hunk 1]
```diff
@@ -42,9 +42,11 @@ const char *const UavcanBarometerBridge::NAME = "baro";
 
 UavcanBarometerBridge::UavcanBarometerBridge(uavcan::INode& node) :
 UavcanCDevSensorBridgeBase("uavcan_baro", "/dev/uavcan/baro", BARO_BASE_DEVICE_PATH, ORB_ID(sensor_baro)),
-_sub_air_data(node),
+_sub_air_pressure_data(node),
```
Comment (davids5) [inline src/modules/uavcan/sensors/baro.cpp]: @LorenzMeier - this is the result of StaticAirData being broken up into  StaticTemperature and StaticPressure.

I am using last_temperature (below) to merge the call back data into the report. Is this acceptable?

Comment (pavel-kirienko) [inline src/modules/uavcan/sensors/baro.cpp]: It is acceptable.

**Comment by davids5** [comment]
@LorenzMeier Since @mcharleb submitted a pr for the params fix - I have removed 8f032a6

I have tested the the ability to bootload the px4cannode-v1 and it it works fine. 

So this it ready to go from my perspective.

**Comment by LorenzMeier** [comment]
@davids5 It looks a bit suspicious like whitespace changes - did you do these on purpose? At any rate please run Tools/fix_code_style.sh and if this was not intended, please make sure to cross-check your editor settings.

**Comment by davids5** [comment]
@LorenzMeier I just re-ran the Astyle fix, unfortunately editor is configured for nuttx coding style. Do you know of a tool or plugin that will switch code styles  in eclipse per directory?

**Comment by LorenzMeier** [comment]
@davids5 @pavel-kirienko I tried to pull this onto the release branch, but ran into a ton of conflicts. I'm not sure how to proceed to get it there - merging this into master now.

**Pull request merged**